app/views/issues/_table.html.erb

```diff
@@ -1,4 +1,4 @@
-<% cache ['issues-table', @all_columns, issues.map(&:id), @issues.map(&:updated_at).map(&:to_i).sort.last, @tags] do %>
+<% cache ['issues-table', @all_columns, @issues.map(&:affected_ids), issues.map(&:id), @issues.map(&:updated_at).map(&:to_i).sort.last, @tags] do %>
```

Comment:
Should issues be `touched` when nodes are updated so we don't have to add this to the cache key?

Reply:
We already have `touch: true` in the `belongs_to` association in the evidence model, and since issues and nodes are connected by evidence, this should suffice in most cases. The reason this is a concern is specific to the merge action, which uses `update_all` so no callbacks are run

Reply:
In order to fix this we could call `update` instead like you mentioned above, but I am concerned about performance implications